pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -722,7 +722,7 @@ stdenv.mkDerivation (finalAttrs: {
     addOpenGLRunpath ${placeholder "lib"}/lib/libavutil.so
   ''
   # https://trac.ffmpeg.org/ticket/10809
-  + optionalString (versionAtLeast version "5.0" && withVulkan) ''
+  + optionalString (versionAtLeast version "5.0" && withVulkan && !stdenv.hostPlatform.isMinGW) ''
```

Comment:
Oh, this should probably be guarded behind isLinux instead as this .so patching is quite linux-specific.